### ISSUE TITLE
cli - Support for  project and cluster lookup by name (currently only look up by id supported)

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11459]
rancher version - v1.0.0-alpha5

Currently there is support for  project and cluster lookup name.
currently only look up by id supported in case of any cluster/project commands that require cluster/project to be passed.

**Comment by deniseschannon** [comment]
due to no name restrictions for project/cluster (aka the fact that there is no unique constraint on names), we'll move this out for now.

**Comment by dramich** [comment]
Available in v1.0.0-alpha13

**Comment by sangeethah** [comment]
Tested with v1.0.0-alpha13

Able to create project in a different cluster (other than default) by specifying "--cluster" option to which cluster name can be passed.

Able to list workloads from project (other than default) by specifying "--project" option to which cluster name  can be passed.

```
 ./rancher project create --cluster santest1do --description "my new cluster" hello3
./rancher ps --project mynewp1
```
